api.c

```diff
@@ -572,10 +571,6 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
-		/* MAXIMUM WRITE SAME LENGTH */
-		val64 = htobe64(VPD_MAX_WRITE_SAME_LENGTH);
-		memcpy(&data[36], &val64, 8);
-
```

Comment:
> Here it using "may set", not a must or should. So I think the "explicitly" you mentioned should be non-zero and a larger WS MAX LENGTH than default value 0xFFFF.

Nice spec find. I agree with you. Let's just return a value larger than 0xffff then.